configs/default/HGLR-HGLRC_F722_AIO_40V1.config

```diff
@@ -0,0 +1,144 @@
+# Betaflight / STM32F7X2 (S7X2) 4.4.0 Feb 15 2023 / 06:10:00 (4605309d8) MSP API: 1.45
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_MAX7456
+
+board_name HGLRC_F722_AIO_40V1
```

Comment:
```suggestion
board_name HGLRCF722AIO
```

Reply:
> 

We will have two FC models with different functions in the future, and we hope to maintain this name to avoid confusion about the target